src/NUnitFramework/framework/Constraints/CollectionEquivalentConstraintResult.cs

```diff
@@ -51,13 +51,10 @@ public class CollectionEquivalentConstraintResult : ConstraintResult
             _tallyResult = tallyResult;
         }
 
-        /// <summary>Write the custom failure message for this object's <see cref="CollectionEquivalentConstraint"/>.</summary>
+         /// <summary>Write the custom additional chainable message to the failure message for a failing constraint. <see cref="CollectionEquivalentConstraint"/>.</summary>
```

Comment:
⚠ This line is indented too far, and we should reflect the name change since we no longer call it ‘chainable message.’
Also, the end of the line renders as, “…for a failing constraint. [CollectionEquivalentConstraint](#example).” I'd be fine removing either `constraint. ` or the `<see>`.

Maybe something like this?


```cs
/// Write any additional lines (following <c>Expected:</c> and <c>But was:</c>) for a failing constraint.
```